2022-08-01 19:14:37,241 INFO    Thread-3  :28568 [wandb_setup.py:_flush():76] Configure stats pid to 28568
2022-08-01 19:14:37,241 INFO    Thread-3  :28568 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-01 19:14:37,241 INFO    Thread-3  :28568 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-01 19:14:37,241 INFO    Thread-3  :28568 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_b', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'sweep_id': 'j85nql5s', 'run_id': 'bbd9a08b', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-j85nql5s/config-bbd9a08b.yaml'}
2022-08-01 19:14:37,241 WARNING Thread-3  :28568 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-01 19:14:37,241 INFO    Thread-3  :28568 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-01 19:14:37,241 INFO    Thread-3  :28568 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_191437-bbd9a08b/logs/debug.log
2022-08-01 19:14:37,242 INFO    Thread-3  :28568 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_191437-bbd9a08b/logs/debug-internal.log
2022-08-01 19:14:37,245 INFO    Thread-3  :28568 [wandb_init.py:init():495] calling init triggers
2022-08-01 19:14:37,245 INFO    Thread-3  :28568 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 128, 'IMS_PER_BATCH': 4, 'iteration': 300, 'learning_rate': 0.0006796266092151828, 'model': 'COCO-Detection/faster_rcnn_R_101_DC5_3x.yaml'}
config: {}
2022-08-01 19:14:37,249 INFO    Thread-3  :28568 [wandb_init.py:init():548] starting backend
2022-08-01 19:14:37,249 INFO    Thread-3  :28568 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-01 19:14:37,270 INFO    Thread-3  :28568 [backend.py:ensure_launched():217] starting backend process...
2022-08-01 19:14:37,278 INFO    Thread-3  :28568 [backend.py:ensure_launched():222] started backend process with pid: 28667
2022-08-01 19:14:37,280 INFO    Thread-3  :28568 [wandb_init.py:init():558] backend started and connected
2022-08-01 19:14:37,294 INFO    Thread-3  :28568 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 128, 'IMS_PER_BATCH': 4, 'iteration': 300, 'learning_rate': 0.0006796266092151828, 'model': 'COCO-Detection/faster_rcnn_R_101_DC5_3x.yaml'}
2022-08-01 19:14:37,297 INFO    Thread-3  :28568 [wandb_init.py:init():634] updated telemetry
2022-08-01 19:14:37,298 INFO    Thread-3  :28568 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-01 19:14:38,072 INFO    Thread-3  :28568 [wandb_run.py:_on_init():1919] communicating current version
2022-08-01 19:14:38,124 INFO    Thread-3  :28568 [wandb_run.py:_on_init():1923] got version response 
2022-08-01 19:14:38,124 INFO    Thread-3  :28568 [wandb_init.py:init():700] starting run threads in backend
2022-08-01 19:14:39,259 INFO    Thread-3  :28568 [wandb_run.py:_console_start():1893] atexit reg
2022-08-01 19:14:39,264 INFO    Thread-3  :28568 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-01 19:14:39,265 INFO    Thread-3  :28568 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-01 19:14:39,265 ERROR   Thread-3  :28568 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-01 19:14:39,267 INFO    Thread-3  :28568 [wandb_init.py:init():732] run started, returning control to user process
2022-08-01 19:24:00,959 INFO    Thread-3  :28568 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_b/bbd9a08b
2022-08-01 19:24:00,960 INFO    Thread-3  :28568 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 0
2022-08-01 19:24:00,965 INFO    Thread-3  :28568 [wandb_run.py:_restore():1834] restore
2022-08-01 19:24:00,968 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 19:24:01,172 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 19:24:01,434 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 19:24:02,390 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 6043
}

2022-08-01 19:24:02,493 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 6043
}

2022-08-01 19:24:02,595 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 6043
}

2022-08-01 19:24:02,699 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 6043
}

2022-08-01 19:24:02,802 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1287
  total_bytes: 6043
}

2022-08-01 19:24:02,904 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2022-08-01 19:24:03,007 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2022-08-01 19:24:03,109 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2022-08-01 19:24:03,212 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2022-08-01 19:24:03,314 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2022-08-01 19:24:04,368 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2022-08-01 19:24:04,911 INFO    Thread-3  :28568 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}
local_info {
}

2022-08-01 19:24:06,043 INFO    Thread-3  :28568 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-01 19:24:06,044 INFO    Thread-3  :28568 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-01 19:24:06,047 INFO    Thread-3  :28568 [wandb_run.py:_footer_sync_info():3055] logging synced files
